### DISCUSSION TITLE
12GB VRAM vs 16GB+ VRAM difference / 4070 Ti Super / 50 Series chat

### DISCUSSION THREAD

**Discussion by magicAUS** [lllyasviel/Fooocus#2710] [General]
I'm going to update my video card as for the life of me can't get my 12GB VRAM AMD 6700 XT to work with this outside of low VRAM mode, along with a third party software continuously crashing with the current AMD GPU.

Is there going to be much difference between 12GB VRAM and 16GB VRAM using Fooocus? I.e. RTX 4070 Super vs RTX 4070 Ti Super.

**Comment by Rogala** [comment]
I have not found a comparison between 4070 and 4070 ti super, I myself want to buy 4060 ti 16GB, I came across an article and video with its testing, it clearly shows how a large number of VRAM helps to speed up the process.

[At the end there is a link to the test 4070 ti super](https://www.msi.com/blog/stable-diffusion-xl-best-value-rtx-graphics-card)

[Second link to a Japanese review between 4060 ti and 4070 super, nothing is clear but the graphs are interesting.](https://www.youtube.com/watch?v=rWpV2ObbPX4)

**Reply by magicAUS** [reply]
Thanks for the links and info. I think I will go for the RTX 4070 Ti Super or wait for 5K series. The price isn't too different to the non Ti in Australia, and the extra CUDA cores over the 4060 Ti for my additional use of gaming, video editing etc. makes it an extra incentive.

It's such a shame AMD GPU's suck so much at non-gaming task compatibility / stability.

**Reply by ILikeToasters** [reply]
And that extra VRAM will probably pay off in the future as the requirements for these models goes up. If I was buying now I would go with the larger VRAM. I have a 3070 with 8GB now and getting me by but waiting to upgrade once SD3 comes out and see what it needs.

**Reply by Rogala** [reply]
Honestly guys, I would slow down a little. Series 5*** will have a GDR7, and maybe a wider tire, and more VRAM. Is it worth spending 900 dollars on a video card now?
And you can add SD to your needs in a year

**Reply by ILikeToasters** [reply]
Oh I am not upgrading because of this stuff. I am actually waiting longer because of it.

**Reply by magicAUS** [reply]
Yeah I think I'll try and hold off until 50 series. After further reading on forums for the third party video editing software, the crashing issues I'm having is also common on Nvidia cards.

**Reply by Irmagon** [reply]
RTX 4070 Ti Super now is best choise speed/price for SD, short of buying used 3090 or 4090

**Reply by Rogala** [reply]
Nobody argues with this, but is it worth it while waiting for series 5***, if you are involved in SD in a financial environment, then yes, but if just for the sake of interest.

In my opinion, 4060 ti 16 GB is what allows you to wait a year.

**Reply by uberneko** [reply]
Hello -
Just stumbled across that benchmark -
https://www.tomshardware.com/pc-components/gpus/stable-diffusion-benchmarks

I also would have thought 16Go instead of 8Go would be better for the 4060 Ti - but here on both tables the 16Go one... performs lower ?!?

**Reply by mashb1t** [reply]
Fyi this benchmark is from December 15, 2023 and does not include SDXL.

> We haven't tested SDXL, yet, mostly because the memory demands and getting it running properly tend to be even higher than 768x768 image generation. 

In general you can future-proof your setup by rather getting a GPU with more VRAM than the one with less while also keeping your requirements on hardware demands (gaming, image generation, web browsing, etc.) in mind.

**Reply by uberneko** [reply]
Thanks !
Ok sorry I'm quite new in AI... so I should push for higher VRAM GPU then -
That's becoming a difficult exercise for me as I also would like to be able to play around still with Blender (I'm much more advanced here) - the requirements AI / 3D do not seem to be totally the same.

**Comment by Rogala** [comment]
[I don't want to create a new topic, all rumors and news about 5000 series.](https://videocardz.com/nvidia/geforce-50)

Key takeaways from this information.
![Снимок экрана 2024-04-08 225658](https://github.com/lllyasviel/Fooocus/assets/7437582/a129f488-f53d-4e46-be6b-5ed040ccc850)
![Снимок экрана 2024-04-08 225715](https://github.com/lllyasviel/Fooocus/assets/7437582/0f76fafb-cc75-4af1-a297-126b197dcd21)

Many people who follow this 5000 topic assume that the 4000 series will soon become more expensive, since its reserves are artificially low and the price will rise.
Nvidia is preparing people to buy the new series.

**Comment by magicAUS** [comment]
> [I don't want to create a new topic, all rumors and news about 5000 series.](https://videocardz.com/nvidia/geforce-50)
> 
> Many people who follow this 5000 topic assume that the 4000 series will soon become more expensive, since its reserves are artificially low and the price will rise. Nvidia is preparing people to buy the new series.

Updated this thread title to include 50 Series chat. :)

**Comment by kalle07** [comment]
I find that 12GB is fine if you only use Ai, be it LLM(GPT) local or text2img.
if you train loras -> 16GB is very useful
i have the 4060Ti (16GB)

**Comment by magicAUS** [comment]
I ended up purchasing a 4070 Ti Super which should arrive late next week.

My MOBO is PCIe 4 (looking like v5 needed for 50 Series), and CPU would be a bottleneck for 50 Series as well.

So I'll keep this setup for 1.5-2 years and review thereafter.

**Comment by barepixels** [comment]
get a used 3090.  it's the best bang for the bux

**Comment by magicAUS** [comment]
> get a used 3090. it's the best bang for the bux

Depends where people are located. In Australia, a used 3090 usually costs more than a new 4070 Ti Super, sometimes $400-1,000 more. 

It also draws a lot more heat and power, so need to consider that when we're talking "bang for buck". And a high amount of system RAM is needed to get the most out of the 24GB VRAM. 

I'm loving the 4070 TiS so far. I'm not using complicated SDXL workflows. The generating and saving time using Fooocus realistic higher res is 9.09 seconds, which I'm content with. Fooocus only seems to use ~7-8.5GB of the 16GB VRAM atm.

**Reply by Rogala** [reply]
[Try disabling general VRAM or adding the `--always-gpu` flag or even both together, I wonder what the VRAM utilization will be then](https://nvidia.custhelp.com/app/answers/detail/a_id/5490/~/system-memory-fallback-for-stable-diffusion)

**Reply by magicAUS** [reply]
Thanks for that. Result summary below using run_realistic, 1472×704, performance quality:

1. With changing the settings in NVIDIA control panel only: Went up to 11.6GB true VRAM utilisation. Generating and saving time: 17.02 seconds

2. then added --always-gpu flag in addition to above: 15.1/16GB true VRAM utilisation. Generating and saving time test 1: 20.91 secs, test 2, 16.9 secs

3. Settings reverted to normal all-around (I.e. removal of 1 & 2 above). 8.3GB true VRAM utilised. Generating and saving time: 17.07 seconds, test 2 Generating and saving time: 16.47 seconds.

I felt it loaded the models and launched the browser app a bit faster in config 1 & 2 above.

I'll try again tomorrow to see if I can replicate single digit second renders in any scenario like I could prior to making changes. But in any case, doesn't seem to be a significant difference in render times regardless whether 8-15GB VRAM is utilised.

**Reply by Rogala** [reply]
It turns out that even with the increased use of VRAM bypassing RAM, the speed is limited by the very power of the video card processor.
Maybe when we move to SD3 there will be more consumption and 16 GB will play its role there.

Try upscaling x2 there should be an increase in the speed and decreases in time of VRAM needs more.

**Reply by magicAUS** [reply]
Not much difference:

Upscale 2x of 1472 x 704 images w/ --always-gpu & NVIDIA control panel setting changed:
15.4GB VRAM utilised
image 1: Generating and saving time: 50.30 seconds
image 2: Generating and saving time: 50.27 seconds

Upscale 2x of 1472 x 704 images w/ Settings reverted to normal all-around:
8.8GB VRAM utilised
image 1: Generating and saving time: 51.29 seconds
image 2: Generating and saving time: 51.07 seconds

**Reply by Rogala** [reply]
You should ask @mashb1t  why there is no acceleration, if everything goes through VRAM and GPU it should be faster.

I don't know much about the intricacies of the job FOOOCUS

**Reply by magicAUS** [reply]
@mashb1t , thoughts? Other PC specs FYI:

i5-12400F 
32GB DDR4 RAM
Win 11 Pro 64-bit 23H2
NVMe SSD

**Reply by mashb1t** [reply]
@magicAUS if usage of either VRAM or RAM doesn't make a difference it's your GPU compute power bottlenecking the process.

**Reply by magicAUS** [reply]
@mashb1t I have an 850W PSU (MSI MAG A850GF), and GPU Power is 260-286 W during rendering (100% power, memory clock ~10,500 MHz). GPU OC software is set to 100% power target. Set vram state to: HIGH_VRAM is displayed.
